### ISSUE TITLE
NSG between web servers and sql servers does not work

### ISSUE THREAD

**Issue by smartao** [microsoft/MCW-Enterprise-class-networking#47]
Hi
I am doing this lab and at least with me the results are not going as expected.

There are two tests to do:
> On the jump host virtual machine (WGMGMT1), initiate a Remote Desktop session to the WGWEB1 via its private IP address (10.8.0.4). This should be successful since it is allowed by Azure Firewall. However, an attempt to connect via Remote Desktop to the WGSQL1 via its private IP address should fail since it is blocked by a network security group.
> 
> On the jump host virtual machine (WGMGMT1), initiate a Remote Desktop session to the WGWEB2 via its private IP address (10.8.0.5). This should be successful since it is allowed by Azure Firewall. However, an attempt to connect via Remote Desktop to the WGSQL1 via its private IP address should fail since it is blocked by a network security group

However, I found that from the WGWEB1 and WGWEB2 servers the RDP access to the WGSQL1 host worked normally.

I started to investigate all the settings and I think I understand why.

1 - NSG WGAppNSG1 is created
2 - It is configuring several INBOUND rules
3 - NSG WGAppNSG1 is associated only with VNET WGVNet2 subnet AppSubnet

NSG is added only to the application server subnet (WGAppNSG1).
To prevent application servers (WGWEB1 and WGWEB2) from accessing the SQL server (WGSQL1) it is necessary to:
or configure an OUTBOUND rule in NSG WGAppNSG1
or else create an NSG just for the data subnet (DataSubnet) blocking the RDP in the INBOUND rules

In my case I was able to block access by creating an OUTBOUND blocking rule in NSG WGAppNSG1
```
Source: Virtual network 
Source port ranges: *
Destination port ranges: 3389
Protocol: Any
Action: Deny
Priority: 100
Name: BlockRDP
```

Please confirm if what I'm talking about is correct or if just some wrong configuration in the way I set up the lab.

Thanks

**Comment by DawnmarieDesJardins** [comment]
@paulopsgility - Hi Paul, please review and advise.

**Comment by paulopsgility** [comment]
@smartao We are in the middle of a test-fix for this one. We'll verify this is working.

**Comment by DawnmarieDesJardins** [comment]
Closing issue as resolved with merge of PR #52
